core/server/themes/to-json.js

```diff
@@ -34,5 +38,10 @@ module.exports = function toJSON(name, checkedTheme) {
         }
     }
 
+    // CASE: if you want a JSON response for a single theme, which is not active.
+    if (_.find(themeResult, {active: true}) && active.get()) {
+        _.find(themeResult, {active: true}).templates = active.get().customTemplates;
```

Comment:
@kevinansfield Should we rename the `templates` property to `customTemplates` or `templates: {custom: ..`?
Because it's a bit hard to read, because we only return the custom templates.

Reply:
Can someone please run the final format / spec of the API endpoints by me today please?